pkgs/development/python-modules/mlx/default.nix

```
@@ -0,0 +1,73 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, buildPythonPackage
+, python3Packages
+, pybind11
+, cmake
+, xcbuild
+, zsh
+, darwin
+, blas
+, lapack
+}:
+
+let
+  # static dependencies included directly during complication
```

```suggestion
  # static dependencies included directly during compilation
```

Except if it was on purpose...